src/map/atcommand.cpp

```diff
@@ -5296,7 +5281,7 @@ ACMD_FUNC(unjail)
 	}
 
 	//Reset jail time to 1 sec.
-	sc_start(NULL,&pl_sd->bl,SC_JAILED,100,1,1000);
+	pc_jail(*pl_sd, 0);
```

Comment:
Guess this one needs special handling? Otherwise it will re-warp the player into jail?